aptos-move/framework/src/natives/aggregator_natives/mod.rs

```diff
@@ -3,7 +3,9 @@
 
 pub mod aggregator;
 pub mod aggregator_factory;
+pub mod aggregator_v2;
 pub mod context;
 mod helpers;
+mod helpers_v2;
```

Comment:
What is the motivation to have V2 for helpers and not place them into `helpers`? `get_aggregator_field` is already defined there, we can add some separation like
```
\\ ------ v1 ------- \\
...
\\ ------ v2------- \\
...
```
Or do you think having 2 files is cleaner?

Reply:
I feel 2 files is cleaner. Other than `get_aggregator_field`, nothing else is common between the two. Aggregator v1 and v2 have different struct format, and thereby different ways of unwrapping the structref.

Reply:
Sounds good :)